tap_github/streams.py

```diff
@@ -372,17 +372,6 @@ def get_records(self, context: Optional[dict] = None) -> Iterable[Dict[str, Any]
 
         return super().get_records(context)
 
-    def get_url_params(
-        self, context: Optional[dict], next_page_token: Optional[Any]
-    ) -> Dict[str, Any]:
-        """Return a dictionary of values to be used in URL parameterization."""
-        params = super().get_url_params(context, next_page_token)
-        if self.replication_key:
-            since = self.get_starting_timestamp(context)
-            if since:
-                params["since"] = since
-        return params
-
```

Comment:
Why was this removal done?

Reply:
I followed the feedback @laurentS has received on the parent repo.